website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Building and Configuring Artifacts"
+title: "Building and configuring artifacts"
 id: "artifacts"
 description: "You can use artifacts to power different aspects of dbt Cloud." 
```

Comment:
Given the limited space available, it feels wasteful to always open with "You can use...". Can we skip to the meaty stuff? (Applies to all of these). I also rewrote this one because "different aspects of dbt Cloud" is very nebulous
```suggestion
description: "Use artifacts to power your automated docs site and source freshness data." 
```